### ISSUE TITLE
v1.0.0: `node_modules/ember-stargate is missing its addon main file`

### ISSUE THREAD

**Issue by johanrd** [simonihmig/ember-stargate#740]
1) `pnpm install ember stargate`
2) `pnpm start`

See error:

```
ember-stargate at app/node_modules/.pnpm/ember-stargate@1.0.0_@babel+core@7.26.0_@glimmer+component@2.0.0_@glimmer+tracking@1.1.2_@gli_lqhosi66cftqdeg562ikiyrkwe/node_modules/ember-stargate is missing its addon main file
```

Could also bee seen on [npm](https://www.npmjs.com/package/ember-headless-table?activeTab=code) that addon-main.cjs is missing from the build: 

<img width="801" alt="Screenshot 2024-11-21 at 22 45 43" src="https://github.com/user-attachments/assets/86821795-39da-4e1b-aeb4-7aa76c4dcff7">

**Comment by basz** [comment]
yep... its true.

**Comment by simonihmig** [comment]
Sorry for the trouble, should be fixed in v1.0.1: https://www.npmjs.com/package/ember-stargate?activeTab=code
